src/darker/__main__.py

```diff
@@ -74,6 +80,11 @@ def format_edited_parts(
             )
             if enable_isort and not edited_linenums and edited == worktree_content:
                 logger.debug("No changes in %s after isort", src)
+                last_successful_reformat = (
+                    src,
+                    worktree_content,
+                    TextDocument.from_lines(""),
+                )
```

Comment:
this was my fix for the false AST non-equivalence thing described in #137, though I make the `chosen` lines empty.

Reply:
An empty `chosen` would indicate that all lines have been deleted.

The fix is going to be merged in #138. Could you wait with this PR and rebase on `master` after that?

Reply:
> An empty chosen would indicate that all lines have been deleted.
Ah, ok, I flipped the "sign" in my head - I thought it was the lines that have changed, so an empty chosen would have been from no changes. Thanks for setting me straight :).

> The fix is going to be merged in #138. Could you wait with this PR and rebase on master after that?

Yep, will do that.